app/src/main/java/moe/apex/rule34/preferences/Pref.kt

```diff
@@ -373,6 +375,8 @@ enum class ImageSource(override val description: String, val site: ImageBoard) :
     SAFEBOORU("Safebooru", Safebooru),
     DANBOORU("Danbooru", Danbooru),
     GELBOORU("Gelbooru", Gelbooru),
+    KONACHAN_SFW("Konachan (SFW)", KonachanSFW),
+    KONACHAN_NSFW("Konachan (NSFW)", KonachanNSFW),
+    R34("Rule34", Rule34),
     YANDERE("Yande.re", Yandere),
-    R34("Rule34", Rule34)
```

Comment:
Revert this please. These are the order they display in the UI and R34 should be the last one (mostly because it's only there for legacy reasons and it discourages use if its at the end). The placements of the Konachan are fine.